### PR TITLE
Add getopt check for db-helper

### PR DESCRIPTION
**Pull request by david-xie**
I added a test for getopt

### CODE REVIEW

File: bin/db-helper [hunk 1]
```diff
@@ -99,7 +110,7 @@ function create_clean_file() {
     local tables=`cat $ddl_file | grep -i "create table" | awk ' { print $3 } '`
 
     local tmp="/tmp/clean-$DATABASE.$$"
-    echo "/*! use $DATABASE; */" >> $tmp
+    echo "/*! USE $DATABASE; */" >> $tmp
```
Comment (sbrossie) [inline bin/db-helper]: That syntax does not seem to work for me (when using mysql on macos). I get the following error:
`ERROR 1064 (42000) at line 1: You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near '' at line 1`

However if i just specify;
`use $database` without the comment that works. I am unclear if this was added for postgreSQL support?

Comment (pierre) [inline bin/db-helper]: It should be `/*! use $DATABASE */;`(semicolon at the end).

Comment (sbrossie) [inline bin/db-helper]: Yes indeed...

Comment (david-xie) [inline bin/db-helper]: Hi @sbrossie and @pierre , I have updated it and got it tested in both MySQL and PostgreSQL en v.

Below are test results:

1. Create tables in MySQL:

	root@f93a63aa0868:/app# bin/db-helper --action create -h mysql -u root -p pass4u
	Applying new schema to database killbill
	Warning: Using a password on the command line interface can be insecure.

2. Clean tables in MySQL:

	root@f93a63aa0868:/app# bin/db-helper -a clean -h mysql -u root -p pass4u
	Cleaning db tables on database killbill
	Warning: Using a password on the command line interface can be insecure.

3. Create tables in PostgreSQL:

	root@e0f5eab1605a:/app# bin/db-helper -a create -u postgres -p pass4u --driver postgres -h postgres
	Applying new schema to database killbill
	NOTICE:  drop cascades to 94 other objects
	DETAIL:  drop cascades to table accounts column created_date
	drop cascades to table accounts column updated_date
	drop cascades to table account_history column created_date
	drop cascades to table account_history column updated_date
	drop cascades to table account_emails column created_date
	drop cascades to table account_emails column updated_date
	drop cascades to table account_email_history column created_date
	drop cascades to table account_email_history column updated_date
	drop cascades to table bus_ext_events column created_date
	drop cascades to table bus_ext_events column processing_available_date
	drop cascades to table bus_ext_events_history column created_date
	drop cascades to table bus_ext_events_history column processing_available_date
	drop cascades to table catalog_override_plan_definition column effective_date
	drop cascades to table catalog_override_plan_definition column created_date
	drop cascades to table catalog_override_phase_definition column effective_date
	drop cascades to table catalog_override_phase_definition column created_date
	drop cascades to table catalog_override_plan_phase column created_date
	drop cascades to table subscription_events column requested_date
	drop cascades to table subscription_events column effective_date
	drop cascades to table subscription_events column created_date
	drop cascades to table subscription_events column updated_date
	drop cascades to table subscriptions column start_date
	drop cascades to table subscriptions column bundle_start_date
	drop cascades to table subscriptions column charged_through_date
	drop cascades to table subscriptions column created_date
	drop cascades to table subscriptions column updated_date
	drop cascades to table bundles column last_sys_update_date
	drop cascades to table bundles column original_created_date
	drop cascades to table bundles column created_date
	drop cascades to table bundles column updated_date
	drop cascades to table blocking_states column effective_date
	drop cascades to table blocking_states column created_date
	drop cascades to table blocking_states column updated_date
	drop cascades to table invoice_items column created_date
	drop cascades to table invoices column created_date
	drop cascades to table invoice_payments column payment_date
	drop cascades to table invoice_payments column created_date
	drop cascades to table payment_attempts column created_date
	drop cascades to table payment_attempts column updated_date
	drop cascades to table payment_attempt_history column created_date
	drop cascades to table payment_attempt_history column updated_date
	drop cascades to table payment_methods column created_date
	drop cascades to table payment_methods column updated_date
	drop cascades to table payment_method_history column created_date
	drop cascades to table payment_method_history column updated_date
	drop cascades to table payments column created_date
	drop cascades to table payments column updated_date
	drop cascades to table payment_history column created_date
	drop cascades to table payment_history column updated_date
	drop cascades to table payment_transactions column effective_date
	drop cascades to table payment_transactions column created_date
	drop cascades to table payment_transactions column updated_date
	drop cascades to table payment_transaction_history column effective_date
	drop cascades to table payment_transaction_history column created_date
	drop cascades to table payment_transaction_history column updated_date
	drop cascades to table _invoice_payment_control_plugin_auto_pay_off column created_date
	drop cascades to table rolled_up_usage column created_date
	drop cascades to table custom_fields column created_date
	drop cascades to table custom_fields column updated_date
	drop cascades to table custom_field_history column created_date
	drop cascades to table custom_field_history column updated_date
	drop cascades to table tag_definitions column created_date
	drop cascades to table tag_definitions column updated_date
	drop cascades to table tag_definition_history column created_date
	drop cascades to table tag_definition_history column updated_date
	drop cascades to table tags column created_date
	drop cascades to table tags column updated_date
	drop cascades to table tag_history column created_date
	drop cascades to table tag_history column updated_date
	drop cascades to table audit_log column created_date
	drop cascades to table notifications column created_date
	drop cascades to table notifications column processing_available_date
	drop cascades to table notifications column effective_date
	drop cascades to table notifications_history column created_date
	drop cascades to table notifications_history column processing_available_date
	drop cascades to table notifications_history column effective_date
	drop cascades to table bus_events column created_date
	drop cascades to table bus_events column processing_available_date
	drop cascades to table bus_events_history column created_date
	drop cascades to table bus_events_history column processing_available_date
	drop cascades to table sessions column start_timestamp
	drop cascades to table sessions column last_access_time
	drop cascades to table users column created_date
	drop cascades to table users column updated_date
	drop cascades to table user_roles column created_date
	drop cascades to table user_roles column updated_date
	drop cascades to table roles_permissions column created_date
	drop cascades to table roles_permissions column updated_date
	drop cascades to table tenants column created_date
	drop cascades to table tenants column updated_date
	drop cascades to table tenant_kvs column created_date
	drop cascades to table tenant_kvs column updated_date
	drop cascades to table tenant_broadcasts column created_date
	drop cascades to table tenant_broadcasts column updated_date
	DROP DOMAIN
	CREATE DOMAIN
	NOTICE:  drop cascades to table tenant_kvs column tenant_value
	DROP DOMAIN
	CREATE DOMAIN
	NOTICE:  drop cascades to 3 other objects
	DETAIL:  drop cascades to table payment_attempts column plugin_properties
	drop cascades to table payment_attempt_history column plugin_properties
	drop cascades to table sessions column session_data
	DROP DOMAIN
	CREATE DOMAIN
	CREATE LANGUAGE
	CREATE FUNCTION
	CREATE FUNCTION
	CREATE FUNCTION
	DROP TABLE
	CREATE TABLE
	CREATE INDEX
	CREATE INDEX
	CREATE INDEX
	DROP TABLE
	CREATE TABLE
	CREATE INDEX
	CREATE INDEX
	DROP TABLE
	CREATE TABLE
	CREATE INDEX
	CREATE INDEX
	CREATE INDEX
	DROP TABLE
	CREATE TABLE
	CREATE INDEX
	CREATE INDEX
	DROP TABLE
	CREATE TABLE
	CREATE INDEX
	CREATE INDEX
	DROP TABLE
	CREATE TABLE
	DROP TABLE
	CREATE TABLE
	CREATE INDEX
	DROP TABLE
	CREATE TABLE
	CREATE INDEX
	DROP TABLE
	CREATE TABLE
	CREATE INDEX
	DROP TABLE
	CREATE TABLE
	CREATE INDEX
	CREATE INDEX
	CREATE INDEX
	CREATE INDEX
	DROP TABLE
	CREATE TABLE
	CREATE INDEX
	CREATE INDEX
	CREATE INDEX
	DROP TABLE
	CREATE TABLE
	CREATE INDEX
	CREATE INDEX
	CREATE INDEX
	CREATE INDEX
	DROP TABLE
	CREATE TABLE
	CREATE INDEX
	CREATE INDEX
	DROP TABLE
	CREATE TABLE
	CREATE INDEX
	CREATE INDEX
	CREATE INDEX
	CREATE INDEX
	CREATE INDEX
	DROP TABLE
	CREATE TABLE
	CREATE INDEX
	CREATE INDEX
	CREATE INDEX
	DROP TABLE
	CREATE TABLE
	CREATE INDEX
	CREATE INDEX
	CREATE INDEX
	CREATE INDEX
	CREATE INDEX
	DROP TABLE
	CREATE TABLE
	CREATE INDEX
	CREATE INDEX
	CREATE INDEX
	CREATE INDEX
	CREATE INDEX
	CREATE INDEX
	DROP TABLE
	CREATE TABLE
	CREATE INDEX
	CREATE INDEX
	DROP TABLE
	CREATE TABLE
	CREATE INDEX
	CREATE INDEX
	CREATE INDEX
	CREATE INDEX
	CREATE INDEX
	DROP TABLE
	CREATE TABLE
	CREATE INDEX
	CREATE INDEX
	DROP TABLE
	CREATE TABLE
	CREATE INDEX
	CREATE INDEX
	CREATE INDEX
	CREATE INDEX
	DROP TABLE
	CREATE TABLE
	CREATE INDEX
	CREATE INDEX
	DROP TABLE
	CREATE TABLE
	CREATE INDEX
	CREATE INDEX
	CREATE INDEX
	CREATE INDEX
	CREATE INDEX
	DROP TABLE
	CREATE TABLE
	CREATE INDEX
	CREATE INDEX
	DROP TABLE
	CREATE TABLE
	CREATE INDEX
	DROP TABLE
	CREATE TABLE
	CREATE INDEX
	CREATE INDEX
	CREATE INDEX
	CREATE INDEX
	DROP TABLE
	CREATE TABLE
	CREATE INDEX
	CREATE INDEX
	CREATE INDEX
	DROP TABLE
	CREATE TABLE
	CREATE INDEX
	CREATE INDEX
	CREATE INDEX
	DROP TABLE
	CREATE TABLE
	CREATE INDEX
	CREATE INDEX
	DROP TABLE
	CREATE TABLE
	CREATE INDEX
	CREATE INDEX
	CREATE INDEX
	CREATE INDEX
	DROP TABLE
	CREATE TABLE
	CREATE INDEX
	CREATE INDEX
	CREATE INDEX
	DROP TABLE
	CREATE TABLE
	CREATE INDEX
	CREATE INDEX
	CREATE INDEX
	DROP TABLE
	CREATE TABLE
	CREATE INDEX
	CREATE INDEX
	CREATE INDEX
	CREATE INDEX
	DROP TABLE
	CREATE TABLE
	CREATE INDEX
	CREATE INDEX
	CREATE INDEX
	CREATE INDEX
	DROP TABLE
	CREATE TABLE
	DROP TABLE
	CREATE TABLE
	CREATE INDEX
	CREATE INDEX
	DROP TABLE
	CREATE TABLE
	DROP TABLE
	CREATE TABLE
	DROP TABLE
	CREATE TABLE
	CREATE INDEX
	DROP TABLE
	CREATE TABLE
	CREATE INDEX
	DROP TABLE
	CREATE TABLE
	CREATE INDEX
	DROP TABLE
	CREATE TABLE
	CREATE INDEX
	CREATE INDEX
	DROP TABLE
	CREATE TABLE
	CREATE INDEX
	DROP TABLE
	CREATE TABLE
	CREATE INDEX

4. Clean tables in PostgreSQL:

	root@e0f5eab1605a:/app# bin/db-helper -a clean -u postgres -p pass4u --driver postgres -h postgres -d killbill
	Cleaning db tables on database killbill
	TRUNCATE TABLE
	TRUNCATE TABLE
	TRUNCATE TABLE
	TRUNCATE TABLE
	TRUNCATE TABLE
	TRUNCATE TABLE
	TRUNCATE TABLE
	TRUNCATE TABLE
	TRUNCATE TABLE
	TRUNCATE TABLE
	TRUNCATE TABLE
	TRUNCATE TABLE
	TRUNCATE TABLE
	TRUNCATE TABLE
	TRUNCATE TABLE
	TRUNCATE TABLE
	TRUNCATE TABLE
	TRUNCATE TABLE
	TRUNCATE TABLE
	TRUNCATE TABLE
	TRUNCATE TABLE
	TRUNCATE TABLE
	TRUNCATE TABLE
	TRUNCATE TABLE
	TRUNCATE TABLE
	TRUNCATE TABLE
	TRUNCATE TABLE
	TRUNCATE TABLE
	TRUNCATE TABLE
	TRUNCATE TABLE
	TRUNCATE TABLE
	TRUNCATE TABLE
	TRUNCATE TABLE
	TRUNCATE TABLE
	TRUNCATE TABLE
	TRUNCATE TABLE
	TRUNCATE TABLE
	TRUNCATE TABLE
	TRUNCATE TABLE
	TRUNCATE TABLE
	TRUNCATE TABLE
	TRUNCATE TABLE
	TRUNCATE TABLE
	TRUNCATE TABLE

**Pull request merged**